.autover/changes/1292d7ce-01b0-41d2-b0a5-58ae76457ced.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "AWS.Deploy.CLI",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Update node version from 18 to 22"
```

Comment:
Add more information on what was updated. This could be confusing because the deploy tool still supports customers deploying using Node18. We did not update the minimum required, just the default for docker images.

Reply:
fixed